assets/app/features/upgrade/steps.factory.spec.js

```diff
@@ -50,13 +50,541 @@ describe('Stepes Factory', function () {
             active: false,
             finished: false
         }
-    ];
+        ],
+        initialStatusData = {
```

Comment:
Do you want to write code to generate test fixtures? How are you going to test this code?